---
date: 2019-06-03
title: Amissah 2017
categories: 
  - Staphylococcus aureus
description: Study on transmission of antibiotic resistant S. aureus in burn patients in sub-Saharan Africa. AMR genes were detected using Abricate
type: Document
related_post: protocol B
comments: true
doi: 10.3389/fmicb.2017.01906
---


## Summary table

|Key|Value|
|----|----|
|category|assembly-based|
|platform|illumina|
|open source|true|
|reproducible|true|
|documented|true|


## Database
This protocol uses Abricate v0.5-dev3 for detection of AMR genes in S. aureus (in burn patients across sub-Saharan Africa). Command for installation of Abricate and its dependencies is available via github (https://github.com/tseemann/abricate).


## Protocol

* Step 1
Genomic DNA was extracted using the Ultraclean microbial DNA isolation kit


* Step 2
DNA libraries were prepared using the Nextera XT v2 kit and sequenced on the Illumina platform using 2 Ã— 250-bp chemistry

* Step 3
The raw reads were assembled into contigs using spades 3.9.0 and mapped to the fully assembled chromosome of S. aureus COL (ST250) (chromosome: CP000046) using snippy v3.21

* Step 4
Antibiotic resistance detection were undertaken with abricate v0.5-dev3.

## Citation

The paper(s) documenting this protocol:

* Amissah NA, Buultjens AH, Ablordey A, et al. [Methicillin Resistant Staphylococcus aureus Transmission in a Ghanaian Burn Unit: The Importance of Active Surveillance in Resource-Limited Settings. Front Microbiol. 2017](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5635451/)
